src/assets/sitemapper.js

```diff
@@ -150,12 +155,13 @@ export default class Sitemapper {
         // The promise resolved, remove the timeout
         clearTimeout(this.timeoutTable[url]);
 
+        console.log(error);
```

Comment:
Is this intentional to log the error or just left over? If intentional you can use console.error(error)

Reply:
Whoops, unintentional. Just removed.